__tests__/app.test.js

```diff
@@ -141,4 +142,43 @@ describe("/api/articles", () => {
         })
     })
 })
-});
\ No newline at end of file
+});
+describe("/api/articles/:article_id", () => {
+    test("PATCH: 200 updates the votes property of the specified article", () => {
+        return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: 1 })
+        .expect(200)
+        .then(({ body }) => {
+            const {article} = body;
+            expect(article.votes).toBe(101)
+        })
+    })
+    test('PATCH: 200 decerements the votes of the property when passed a negatibe number', () => {
+        return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: -100 })
+        .expect(200)
+        .then(({ body }) => {
+            const { article } = body;
+            expect(article.votes).toBe(0)
+        })
+    });
+    test('PATCH: 404 responds with an error message if the article ID isnt found', () => {
+        return request(app)
+        .patch("/api/articles/700")
+        .send({ inc_votes: 10 })
+        .expect(404)
+        .then(({ body }) => {
+            expect(body.msg).toBe("Article Not Found")
+        })
+    });
+    test("PATCH: 400 responds with an error message if the article ID is invalid", () => {
+        return request(app)
+        .patch("/api/articles/invalidarticletest")
+        .expect(400)
+        .then(({ body }) => {
+            expect(body.msg).toBe("Bad Request Invalid Article ID")
+        })
```

Comment:
There a couple more tests to do here -
Status 400 - incorrect body (ie, property is not a number)
Status 200 - if inc votes is missing should not update the article and instead return the original response